src/worldofzuul/Item.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author mikkellarsen
+ */
+public class Item {
+    /**
+     * Creating private variables 
+     * String which sets the name
+     * Boolean expression which is true by default
+     */
+    private String name;
+    private boolean moveable = true;
```

Comment:
The javadoc should be over the individual attribute, splitting the comment up

Reply:
Fixed a8372db4ab2538105623c5f4f07b351ec0308ed6